### ISSUE TITLE
King County COVID19 - Bug: YouTube video accessibility

### ISSUE THREAD

**Issue by missmatsuko** [a11ySea/testing#19]
## Issue Summary

Some issues related to the YouTube video embedded on [the King County Coronavirus Disease 2019 (COVID-19) page](https://kingcounty.gov/depts/health/covid-19.aspx).

<img width="878" alt="Screenshot of COVID-19: Stay Home YouTube video on the King County COVID-19 page." src="https://user-images.githubusercontent.com/17884198/80635565-e687f100-8a10-11ea-81fe-5c60e1028446.png">

1. Video lacks voiceover or captioning, so blind and screen reader users may not be able to access the content. Most of the information is in the text directly to the right of the video, but not all. 

2. Some parts of the text embedded in the video have insufficient contrast. For example, this white on yellow and yellow and white text reading "Together, we can slow the spread of COVID-19". The yellow is #EED146 and white is #FFFFFF. The contrast is about 1.51:1, failing WCAG contrast requirements.
<img width="653" alt="Screenshot of a frame of the YouTube video, with text that reads 'Together, we can slow the spread of COVID-19'." src="https://user-images.githubusercontent.com/17884198/80635708-1f27ca80-8a11-11ea-871a-63d788eb4c43.png">

## Steps to reproduce

1. Go to [the King County Coronavirus Disease 2019 (COVID-19) page](https://kingcounty.gov/depts/health/covid-19.aspx).
2. Play the COVID-19: Stay Home YouTube video, which is located at the beginning of the page content.
3. Note colour contrast issues, lack of voice-over, lack of captions.

## Behavior
The full video content may not be accessible for people with visual disabilities.

### Expected behavior
The full video content should be accessible for people with visual disabilities.

### Actual behavior
Video has contrast issues and lacks captions and voice-over.

### Device Information:

 - OS: [e.g. Windows, Mac iOS]

### Screen Reader or AT Information

 - [e.g. JAWS 2020.2003.13]
 - [e.g. NVDA 2019.3.1]

### Current Code
```
(code snippet)
```

### Suggested Code
```
(code snippet)
```

# Specifications

- **WCAG Principal:**
- **WCAG SC:**
- **Severity:** Fairly minor, since most of the information is in text next to the video and the rest isn't super critical.

## Additional References

- Link to any articles or specs that give further guidance

**Comment by alboss** [comment]
We will pass this along to the content creators for Public Health. Changing a video after the fact is a challenge even when one isn't in the throes of a  pandemic, so I don't know that we will be able to fully fix the issues identified--but it's great information for them to have for the future.
